common/remote_util.h

```diff
@@ -92,15 +93,18 @@ class RemoteUtil {
 
   // Runs |remote_command| on the remote device. The command must be properly
   // escaped. |name| is the name of the command displayed in the logs.
-  absl::Status Run(std::string remote_command, std::string name);
+  absl::Status Run(std::string remote_command, std::string name,
+                   ArchType arch_type);
 
   // Same as Run(), but captures both stdout and stderr.
   // If |std_out| or |std_err| are nullptr, the output is not captured.
   absl::Status RunWithCapture(std::string remote_command, std::string name,
```

Comment:
It's a bit non-obvious which arch type need to be passed here (the current machine one or the target machine). Can you please add a description here? (also maybe name a parameter corresponding local_arch_type)